Unweldify lesson1

Unweldify opened the pull request:
Не совсем по "примерной инструкции".... И опыта у меня нет...
Сделал форк репозитория и пр к вам на main потому что на push ветки у меня нет прав.
Предпологаю что вы в силах сделать мою ветку в вашем репозитории и переадресовать мой пулл туда
Напишите пожалуйста либо в комментарии либо мне в личку если я что то упустил и сделал не как надо.

Однако задание я сделал.

Unweldify committed:
Final code, probably not accurate because the task
wasn't sent

Unweldify committed:
Final solution.

Unweldify committed:
Final solution (changed the in_nums from the repo)

Unweldify committed:
Merge branch 'evgeniy-barsukov:main' into Unweldify_lesson1

evgeniy-barsukov commented on main.py:
```diff
@@ -0,0 +1,32 @@
+
+# 
+
+with open("in_nums.txt", "r") as inFile:
```
Здесь и в других местах.
Необходимо следовать стандартам языков. 
РЕР8 говорит, что имена переменных, состоящих более чем из одного слова, разделяются подчеркиванием.
То есть не 'inFile' a 'in_file'

Unweldify replied:
По стандарту PEP8, camelCase является допустимым способом наименования переменных, так же как и snake_case.
Однако, для дальнейшего удобства я изменил названия переменных под ваше предпочтение.

Unweldify committed:
Changed variable names to snake_case
Added type annotations for function arugments and returns
Cleaned up spacing of the code

Unweldify committed:
Merge branch 'Unweldify_lesson1' of https://github.com/Unweldify/first_steps into Unweldify_lesson1

evgeniy-barsukov commented on main.py:
```diff
@@ -0,0 +1,30 @@
+        return get_fibonacci(n - 1) + get_fibonacci(n - 2)
+
+
+def is_prime(n: int, i=2) -> bool:
```
i=2 - должны быть пробелы с обеих сторон от "="

evgeniy-barsukov reviewed (approved):
Несмотря на замечание - Аппрув получен, можешь выполнять слияние
